GameServer/skillhandler/StagAbilityHandler.cs

```diff
@@ -41,7 +41,7 @@ public class StagAbilityHandler : IAbilityActionHandler
 		/// <summary>
 		/// The ability reuse time in milliseconds
 		/// </summary>
-		protected const int REUSE_TIMER = 60000 * 20;
+		protected const int REUSE_TIMER = 60000 * 10; // clait: 10 minutes [og: 20]

```

Comment:
You do not need to put your name in the comments ;) `git blame` is here for that exact purpose ;)